dockerfiles/che/entrypoint.sh

```diff
@@ -322,10 +322,15 @@ add_che_cert_to_truststore() {
   fi
 }
 
-add_public_cert_to_truststore() {
+add_public_certs_to_truststore() {
   JAVA_TRUST_STORE=/home/user/cacerts
+  DEFAULT_JAVA_TRUST_STORE=${JAVA_HOME}/lib/security/cacerts
   DEFAULT_JAVA_TRUST_STOREPASS="changeit"
 
+  if [ ! -f "$JAVA_TRUST_STORE" ]; then
```

Comment:
I think for folder should be '-d' instead of '-f'...

Reply:
No, it is a file